fuel-asm/src/args.rs

```diff
@@ -252,7 +252,7 @@ crate::enum_try_from! {
         PolicyTypes = 0x500,
 
         /// Set `$rA` to `tx.policies[0x00].gasPrice`
-        PolicyGasPrice = 0x501,
+        PolicyTip = 0x501,
```

Comment:
maybe we should call this "GasTip" in case we have other unit dimensions in the future.

Reply:
Do we want the tip denominated in gas or in the base asset?